ajax/libs/unibox/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "unibox",
+  "title": "unibox - search suggestion",
+  "filename": "js/unibox.min.js",
+  "description": "A powerful search suggestion input box.",
+  "keywords": [
+    "search",
+    "suggestion"
+  ],
+  "version": "1.12.0",
+  "author": {
+    "name": "David Urbansky",
+    "url": "https://github.com/ddsky/unibox"
+  },
+  "license": "MIT",
+  "homepage": "https://spoonacular.com/meal-planner",
+  "dependencies": {
+    "jquery": ">=1.5"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ddsky/unibox.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ddsky/unibox.git",
+    "basePath": "",
+    "files": [
+      "(css|js)+/unibox*.+(js|css|map)"
```

Comment:
So you defined or invented your own syntax? I can't see it in https://github.com/isaacs/node-glob